src/display/api.js

```diff
@@ -143,6 +152,19 @@ function setPDFNetworkStreamFactory(pdfNetworkStreamFactory) {
  *   reading built-in CMap files. Providing a custom factory is useful for
  *   environments without Fetch API or `XMLHttpRequest` support, such as
  *   Node.js. The default value is {DOMCMapReaderFactory}.
+ * @property {boolean} [useSystemFonts] - When true, fonts that aren't embedded
+ *   in the PDF will fallback to a system font. Defaults to true for web
+ *   environments and false for node.
+ * @property {string} [standardFontDataUrl] - The URL where the standard font
+ *   files are located. Include the trailing slash.
+ * @property {boolean} [standardFontDataWorkerFetch] - Enable fetching the font
```

Comment:
@Snuffleupagus You had mentioned using worker fetch for cmaps too. What are you thoughts on changing this to `useWorkerFetch`, so we just have one option to enable it?

Reply:
> What are you thoughts on changing this to `useWorkerFetch`, so we just have one option to enable it?

That's a really *excellent* idea, the fewer new options we can add the better in my opinion (for general testability and maintainability reasons).